docs/2.guide/1.concepts/3.rendering.md

```diff
@@ -89,7 +89,8 @@ Nuxt 3 starting from rc.12 comes with the public beta for route rules and hybrid
 - `ssr` - Disables server-side rendering for sections of your app and make them SPA-only with `ssr: false`
 - `cors` - Automatically adds cors headers with `cors: true` - you can customize the output by overriding with `headers`
 - `headers` - Add specific headers to sections of your site - for example, your assets
-- `static` and `swr` - `static` enables a single (on-demand) build; `swr` enables a static build, that lasts for a configurable TTL. (currently enables full incremental static generation on Netlify, with Vercel coming soon)
+- `swr`: Server-side renders sections of your app and add cache headers in the response for the server or the reverse proxy to cache it for a configurable TTL. For Netlify and Vercel, the response is also added to the CDN layer. Note that the node-server preset of Nitro is able to cache the response.
+- `static`: Enables a single (on-demand) build (currently enables full incremental static generation on Netlify, with Vercel coming soon)
```

Comment:
swr and static have the same behaviour of on-demand generation - it's just that static doesn't revalidate after the first request. We could add `prerender` also as another route rule option.

Reply:
It's very confusing to call it `static` then IMHO. I will try to modify this part too.

Reply:
Are you sure it's the case? In node the only header I see is `Cache-Control: public` without `max-age`.

Reply:
It's somewhat more complicated, but it's definitely not _just_ adding headers. Some routes are cached internally by handler; others aren't. See:

https://github.com/unjs/nitro/blob/2c52c62f1e4adbce4f57e28ec1d4929cd2051928/src/runtime/cache.ts#L160-L333

https://github.com/unjs/nitro/blob/ed3a48026e8a27ecfb86e406ba39bdeb418f8573/src/runtime/app.ts#L55-L64

Reply:
Static is more a permanent cache strategy per deployment.

Reply:
~> https://github.com/unjs/nitro/pull/906